code/components/font-renderer/src/GtaGameInterface.cpp

```diff
@@ -287,6 +288,17 @@ static InitFunction initFunction([] ()
 	static std::random_device random_core;
 	static std::mt19937 random(random_core());
 
+	static bool shouldDraw = false;
+
+	Instance<ICoreGameInit>::Get()->OnGameRequestLoad.Connect([]()
```

Comment:
![Having to rewrite the HUD because of this crap...](https://user-images.githubusercontent.com/5144843/61593203-d9fe7e80-abdc-11e9-8d93-c8b015ec8c00.png)

Reply:
This watermark is just annoying as fuck

Reply:
@Subtixx
> This watermark is just annoying as fuck

Other than the fact that a line comment on a pull request from 2 years ago isn't a good place to provide feedback, what's so 'annoying as fuck' about it?

Reply:
A function to let server owners decide on if they want to show it or not. The screenspace is valuable, and having the watermark means 1 less space to put information. I came across this PR when googling how to disable it.

Reply:
You can still 'put information' in any corner, the watermark doesn't take up some magical 'space' as it's semi-transparent.

'Disabling' it is also not going to happen, it's there for fairly evident recognizability reasons, and other games also tend to have a persistent on-screen overlay as well these days.

Reply:
"Other software is doing the same" is a pretty garbage standard to develop by imo.

Reply:
Which is why that's not a sole reason to develop by. Locking this thread now, before this derails further.